### DISCUSSION TITLE
[Accordion] Prevent animation for default open item

### DISCUSSION THREAD

**Discussion by trangcongthanh** [radix-ui/primitives#934] [Help]
How can I prevent the default open item run the animation on the first render? IMO, it should run only when the value changes.
When I have some items with default open state, my layout jumps every time it is mounted.

**Comment by benoitgrelard** [comment]
hi @trangcongthanh yes I think I agree with you, we've had a chat about it in the past and we need to look into how we can do this. We'll keep you updated.

**Comment by jjenzz** [comment]
I've created a PR to prevent mount animation when open by default here https://github.com/radix-ui/primitives/pull/977
